src/scripts/main.js

```diff
@@ -1,3 +1,54 @@
 'use strict';
 
 // write code here
+
+const employees = document.querySelectorAll('li');
+const listEmployees = document.querySelector('ul');
+
+function getSalary(element) {
+  const salary = element.dataset.salary;
+
+  if (salary) {
+    let cleanedSalary = salary.slice(1);
+
+    cleanedSalary = cleanedSalary.split(',').join('');
+
+    return Number(cleanedSalary);
+  }
+
+  return 0;
+}
+
+function sortList(list) {
+  const listItem = [...list];
+
+  return listItem.sort((a, b) => getSalary(b) - getSalary(a));
+}
+
+listEmployees.innerHTML = '';
```

Comment:
You're clearing the innerHTML of the list before sorting and appending the elements. This would remove any event listeners attached to these elements. Instead of clearing the innerHTML, you could directly append the sorted elements. The browser will automatically remove them from their current position before appending them to the new position.